maps/away/ships/yacht_civ/yacht_civ_ghostroles.dm

```diff
@@ -33,4 +33,7 @@
 
 /obj/item/card/id/yacht_civ
 	name = "\improper Civilian yacht identification card"
-	access = list(ACCESS_EXTERNAL_AIRLOCKS)
+
+
+/obj/outfit/admin/yacht_civ/get_id_access()
+	return list(ACCESS_EXTERNAL_AIRLOCKS)
```

Comment:
when I initially added the Yacht, I did not give then IDs cause I thought they wouldn't really need them for anything

right now they get IDs specifically for airlocks

it would be better to just remove access requirements from airlocks, and then remove their IDs

Reply:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/862945e3-e117-474e-999e-d5be5782fc61)

Reply:
removing access requirements from airlocks would be like this

Reply:
I made the changes you requested and it works properly.

At first I had thought it might be more efficient to set the null access on /obj/effect/shuttle_landmark/yacht_civ/dock and let the individual landmarks inherit the values but it doesn't work for some reason so I set each individually.